### ISSUE TITLE
Is there any scripts relatable to training are available? 

### ISSUE THREAD

**Issue by InfiniteLife** [isl-org/DPT#54]
I've seen few issues where people were trying to train the network, but cant find any training code. Is there anything available? I'm interested in loss function, pipeline for data and depth preparation for loss function, it seems rather complicated from reading the paper.

**Comment by younesbelkada** [comment]
Hi! together with my colleague we made an attempt to reproduce the training script here: https://github.com/antocad/FocusOnDepth check it out if you are interested!
